### PR TITLE
json_formatter

### LINKED ISSUE #22: Work on JSON schema
Database for storing user-related progress and data

### PR DESCRIPTION
**Pull request by arahim2**
consists of a writer.py and a test.py file
includes code that is able to create, update and format data in the json format agreed upon in attached document.

The test file runs the writer.py functions and writes the formatted json text in a text file called user.txt
Note: The code is not able to extract and work with stored json variables eg: the overall stock quantity does not change with each transaction.

[json_struct.txt](https://github.com/CS196Illinois/Finance/files/4433082/json_struct.txt)

### CODE REVIEW

**Comment by arahim2** [comment]
Waiting on approval
run test.py for output

File: json_formatter copy/write.py
Comment (MattGeimer) [inline json_formatter copy/write.py:26]: Generally, JSON layouts should not have ":" in them as it sometimes messes up the parser. Given that the stocks are already in the portfolio array, I think it's fine to assume that they're stocks (if we add support for options, futures, etc later, we can write a small script to migrate everything in portfolio into a stocks sub-array).

**Review by MattGeimer — COMMENTED** [review]
Overall, really well done. Just fix the stock key as suggested below, and then I'll merge it.

**Review by avmehta2 — APPROVED** [review]

**Pull request merged**
